src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -671,13 +778,20 @@ def __call__(
             list of `bool`s denoting whether the corresponding generated image likely represents "not-safe-for-work"
             (nsfw) content, according to the `safety_checker`.
         """
+
+        # TODO: add conversion image array to ControlNetConditions
+        if controlnet_conditions is None:
+            controlnet_conditions = [ControlNetCondition(image=image, scale=controlnet_conditioning_scale)]
```

Comment:
Let's maybe also allow to cor
```suggestion
            # let's split images over controlnets
            image_per_control = 1 if isinstance(self.controlnet, ControlNetModel) else len(self.controlnet.nets)
            if image_per_control > 1 and not isinstance(image, list):
                raise ValueError(...)
                
            if len(image) % image_per_control != 0:
                raise ValueError(...)
            
            images = [image[i:i+num_image_per_control] for i in range(0, len(image), image_per_control)]
            controlnet_conditions = [ControlNetCondition(image=image, scale=scale) for image, scale in zip(images,  controlnet_conditioning_scale)]
```

Reply:
Let's also allow multi-controlnet using just `image` and `controlnet_conditioning_scale` input :-)

Reply:
Thanks!